server/rpc/common.go

```diff
@@ -210,16 +210,15 @@ func buildGetBlockVerboseResult(s *Server, block *util.Block, isVerboseTx bool)
 	params := s.cfg.DAGParams
 	blockHeader := block.MsgBlock().Header
 
-	// Get the block chain height.
-	blockChainHeight, err := s.cfg.DAG.BlockChainHeightByHash(hash)
+	blockBlueScore, err := s.cfg.DAG.BlueScoreByBlockHash(hash)
 	if err != nil {
-		context := "Failed to obtain block height"
+		context := "Could not get block blue score"
 		return nil, internalRPCError(err.Error(), context)
 	}
 
 	// Get the hashes for the next blocks unless there are none.
 	var nextHashStrings []string
-	if blockChainHeight < s.cfg.DAG.ChainHeight() { //TODO: (Ori) This is probably wrong. Done only for compilation
+	if blockBlueScore < s.cfg.DAG.SelectedTipBlueScore() {
```

Comment:
This is a somewhat backwards way to check if block has no children.
Why not call ChildHashesByHash and see if it returned anything?
I don't think it does any heavy operations.